docs/user/project-setup.md

```diff
@@ -71,6 +81,23 @@ Add these entries to your `.gitignore` to exclude erk's temporary and session-sp
 - **`.erk/scratch/`**: Session-specific scratch storage. Each Claude session creates temporary files here scoped by session ID. These are ephemeral and should not be committed.
 - **`.impl/`**: Implementation plan files created per-worktree. These track in-progress work and are deleted after successful PR submission.
 
+## Step 5: Commit Your Setup
+
+After completing the setup, commit the following files to git:
+
+```bash
+git add erk.toml .claude/ .gitignore
+git commit -m "Configure erk for project"
+```
+
+**What gets committed:**
+
```

Comment:
remove these commit instructions. just say you should commeit the follow files

Reply:
Resolved via /erk:pr-address at 2025-12-12 05:38 AM EST

_Addressed via `/erk:pr-address` at 2025-12-12 05:38 EST_